src/middlewares/auth.middleware.js

```diff
@@ -0,0 +1,30 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { jwtService } from '../services/index.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'];
```

Comment:
Consider using `req.headers.authorization` instead of `req.headers['authorization']` for consistency and readability.